### ISSUE TITLE
Using CSS letter-spacing to control spacing within a word

### ISSUE THREAD

**Issue by vikassharma1112** [w3c/wcag#918]
Can you please help me for this error?

**Comment by awkawk** [comment]
You're going to need to give us more context.

**Comment by vikassharma1112** [comment]
Hello,

My web page link- http://badlapurseva.com/13/survivor-research
I am testing for WCAG 2.1 AA compliance. - https://mauve.isti.cnr.it/

I have checked my link in WCAG 2.1 AA and it is showing following error "Using CSS letter-spacing to control spacing within a word".

Can you please help me!

**Comment by jake-abma** [comment]
Hi @vikassharma1112 

I don't see "Using CSS letter-spacing to control spacing within a word" when validating the result of http://badlapurseva.com/13/survivor-research in MAUVE but do see other results for text spacing.
The results of MAUVE should be checked manually also as I'm not sure about the exact test rules and if they are valid or might also have some false positives. Do you also check with other 'known' testing tools to compare results?

For text spacing there's a success criteria: https://www.w3.org/WAI/WCAG21/Understanding/text-spacing and also two techniques:

- https://www.w3.org/WAI/WCAG21/Techniques/css/C36
-  https://www.w3.org/WAI/WCAG21/Techniques/css/C35

Please read through these as they may have the answer for you.

Cheers,
Jake

**Comment by mraccess77** [comment]
My personal guess is that a tool is seeing the use of letter spacing as possible failure.  That's not a fail by itself of the current guidelines.  What is failure is if when the text spacing is increased to the set amount the content is cutoff or overlaps, etc.  So the real test is to determine if the containers can hold the text with the spacing applied -- this is generally a manual test at this point.

**Comment by JAWS-test** [comment]
The tool finds 66 violations of SC [1.4.12](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing) and refers to [C8](https://www.w3.org/WAI/WCAG21/Techniques/css/C8).

**Comment by patrickhlauke** [comment]
@alastc I believe this issue can be closed?

**Comment by alastc** [comment]
@patrickhlauke - yep, thanks.
